runner.py

```diff
@@ -266,14 +267,14 @@ def update_and_insert_specs(self):
 
             machine_specs.update(machine_specs_root)
 
-        # Insert auxilary info for the run. Not critical.
+        # Insert auxiliary info for the run. Not critical.
         DB().query("""UPDATE projects
             SET machine_specs=%s, measurement_config=%s, usage_scenario = %s, last_run = NOW()
             WHERE id = %s
             """, params=(
             json.dumps(machine_specs),
```

Comment:
Please also escape the machine_specs as someone could inject an XSS through adding parameters to the process that are html and then these would be dumped here and also rendered into the frontend